__tests__/app.test.js

```diff
@@ -285,4 +286,39 @@ describe("PATCH /api/articles/:article_id", () => {
                 })
             })
     });
+    test('responds with an appropriate status: 400 and error message when provided with a bad votes object (NaN)', () => {
+        return request(app)
+            .patch("/api/articles/9")
+            .send({
+                inc_votes: "100 more votes"
+            })
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe('Bad Request');
+            });
+    });
+    test('responds with an appropriate status: 404 and error message when given a valid but non-existent id', () => {
+        const newVotes = {
+            inc_votes: -100
+        }
+        return request(app)
+            .patch("/api/articles/999")
+            .send(newVotes)
+            .expect(404)
+            .then((response) => {
+                expect(response.body.msg).toBe('Article Does Not Exist');
+            });
+    });
+    test('responds with an appropriate status: 400 and error message when given an invalid id', () => {
+        const newVotes = {
+            inc_votes: -100
+        }
+        return request(app)
+            .patch("/api/articles/not-an-article")
+            .send(newVotes)
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe('Bad Request');
+            });
+    });
```

Comment:
You've not actually tested for if it can increase votes - I woudl suggest that should be your first test.